### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by alexey-sidnev**
* Add a link to TensorFlow GPU pre-requisites
* Minor fixes

### CODE REVIEW

**Comment by alexey-sidnev** [comment]
@alexey-trushkov , could you please take a look?

**Comment by alexey-trushkov** [comment]
@alexey-sidnev looks good, but could you, please,  also fix my remark.

**Comment by alexey-sidnev** [comment]
@alexey-trushkov , I don't see any remarks :(
Don't you forget to press the submit button during the review?

File: README.md [hunk 1]
```diff
@@ -61,10 +64,12 @@ export OPEN_VINO_DIR=<PATH_TO_OPENVINO>
 nosetests
 ```
     
-Note: if you have install the OpenVino toolkit after creating a virtual environment then you have to [recreate one](#create_venv) to install required packages for the Model Optimizer into one.
+Note: if you have install the OpenVino toolkit after creating
```
[the remark this answers is not in the document]
Comment (alexey-sidnev) [reply README.md]: Fixed!

**Pull request merged**